.github/workflows/pr-requirements-check.yml

```
@@ -0,0 +1,101 @@
+      - 'hotfix/**'
+      - 'users/stephen/support-docs-layout-changes'
+    types: [opened, edited]
+  merge_group:
+    branches:
+      - dev
+
+jobs:
+  check-pr-requirements:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)